www/js/app.js

```diff
@@ -1111,6 +1111,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                 settingsStore.setItem("lastSelectedArchive", archiveDirectory, Infinity);
                 // The archive is set : go back to home page to start searching
                 $("#btnHome").click();
+            // callbackError which is called in case of an error
```

Comment:
I would find it more clear to put this comment one line below (inside the anonymous function)